packages/mds-transactions-api/middleware/transaction-api-error.ts

```diff
@@ -0,0 +1,28 @@
+import log from '@mds-core/mds-logger'
+import { isError, isServiceError } from '@mds-core/mds-service-helpers'
+import { BadParamsError, ConflictError, NotFoundError, ServerError, ValidationError } from '@mds-core/mds-utils'
+import express from 'express'
+import HttpStatus from 'http-status-codes'
+
+import { TransactionApiRequest, TransactionApiResponse } from '../@types'
+
+export const TransactionApiErrorMiddleware = (
+  error: Error,
+  req: TransactionApiRequest,
+  res: TransactionApiResponse,
+  next: express.NextFunction
+) => {
+  log.error(req.method, req.originalUrl, error)
+
+  if (isError(error, ValidationError)) return res.status(HttpStatus.BAD_REQUEST).send({ error })
+
+  if (isError(error, BadParamsError)) return res.status(HttpStatus.BAD_REQUEST).send({ error })
```

Comment:
Just confirming, both of these first two cases are `BAD_REQUEST` responses?
If so, does it make sense to consolidate into one case?
``` if (isError(error, ValidationError) || isError(error, BadParamsError)) do stuff```

Reply:
Yup, should both be BAD_REQUEST responses. Could definitely consolidate into one case; it used to be more onerous prior to `isError` existing, but should be fine now

Reply:
Resolved in 33296d3